pychunkedgraph/app/segmentation/common.py

```diff
@@ -1196,3 +1200,18 @@ def operation_details(table_id):
                 details[_k] = _v
         result[int(k)] = details
     return result
+
+
+### DELTA ROOTS ------------------------------------------------------------
+
+
+def delta_roots(table_id):
+    current_app.table_id = table_id
+
+    timestamp_past = _parse_timestamp("timestamp_past", None, return_datetime=True)
```

Comment:
how is this interaction with `default_timestamp=None` and the try clause + `BadRequest` raise in  `_parse_timestamp` supposed to play out?

Reply:
ah good catch.. i think i've fixed it